fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraRepositoryImpl.java

```diff
@@ -220,6 +221,33 @@ public FedoraObject createObject(final String path) throws FedoraException {
     }
 
     @Override
+    public FedoraObject createObject() throws FedoraException {
```

Comment:
Two use cases:  
1.  Create a new object (at the repository root) that has a repository-assigned id. (ie /random-id/)
2.  Create a new object (contained by some other object) that has a repository-assigned id. (ie /specific-path/random-id/)

Maybe the second case is foolish, because it retains some notion that the repository space isn't flat, but it's really convenient from my point of view to be able to stash a whole set of new resources in a container (ie, "migrated-fedora-3-content")